src/TaskList.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useRef } from "react";
+function TaskList(props) {
+  const liRef = useRef(null);
+
+  const handleListChange = () => {
+    if (liRef.current) {
+      liRef.current.classList.toggle("setTachado"); // Alterna la clase "tachado"
```

Comment:
Be consistent. Use names in english.

Remove unnecesarry comment. I know what a toggle does.